cli/types/cypress.d.ts

```diff
@@ -5498,6 +5498,8 @@ declare namespace Cypress {
     message: any
     /** Set to false if you want to control the finishing of the command in the log yourself */
     autoEnd: boolean
+    /** Set to false if you want to control the finishing of the command in the log yourself */
```

Comment:
what's the difference b/t this and autoEnd? same comment

i think you can also update this in `packages/driver/types/internal-types.d.ts` if you don't want it to be public

Reply:
From what I can tell, `autoEnd` is redundant and we don't actually use it at all. The only instance of it is here:

https://github.com/cypress-io/cypress/blob/e0c385e0a2b5c0d343399280ab8b7e2aa2e12113/packages/driver/src/cy/commands/debugging.js#L58-L61

However, using `autoEnd` requires calling `log.finish()`:

https://github.com/cypress-io/cypress/blob/e0c385e0a2b5c0d343399280ab8b7e2aa2e12113/packages/driver/src/cypress/log.js#L443-L451

But I can't find any instances of calling `log.finish()` and the instance from above that uses `autoEnd: true` calls `log.end()`, which is the imperative way to end the snapshot, indicating it was never the intention to "auto-end" the log for `cy.pause()`:

https://github.com/cypress-io/cypress/blob/e0c385e0a2b5c0d343399280ab8b7e2aa2e12113/packages/driver/src/cy/commands/debugging.js#L74-L76

The `autoEnd` option can probably be removed entirely, but that's outside the scope of this PR. I added the `end` option to the types since it's the one we actually use around the codebase for auto-ending a log. I guess this was the first time it was used in TypeScript though, so we hadn't noticed the omission before.